src/main/java/com/miraisolutions/xlconnect/utils/RPOSIXDateTimeFormatter.java

```diff
@@ -153,24 +158,21 @@
               // the 2004 POSIX standard, but it does also say �it is expected
               // that in a future version the default century inferred from a
               // 2-digit year will change�.
-              builder.appendTwoDigitYear(1968, true);
+              builder.appendValueReduced(ChronoField.YEAR, 2, 2, 1969);
```

Comment:
Is the 1969 insteadof 1968 intended?

Reply:
Yes, the range implied by the base year is inclusive, this gives the range 1969-2068 as specified by the comment (unchanged). 1968 would imply the range 1968-2067.

Reply:
Checked that the [R documentation|https://stat.ethz.ch/R-manual/R-devel/library/base/html/strptime.html] as of this comment, still describes the same interval (and R 3.6.1 also parses dates according to it).